### DISCUSSION TITLE
Problem install extension

### DISCUSSION THREAD

**Discussion by alexdesousapereira** [AltimateAI/vscode-dbt-power-user#409] [Q&A]
I am having difficulties configuring the dbt Power User plugin in Visual Studio Code. I have tried following the instructions, but I still can't get the plugin to work properly. The extension is useful for working with dbt more efficiently, but I am having trouble configuring.

I have using dbt-mysql

This is my configurations.

![image](https://user-images.githubusercontent.com/79751281/233112130-d7a1a88c-23bd-42ab-885f-d1b34f666382.png)

**Comment by mdesmet** [comment]
As you can see the env config is greyed out. You need to choose your os, for example` terminal.integrated.env.osx`.

**Comment by anandgupta42** [comment]
@alexdesousapereira Is your problem resolved?
